pandamium_datapack/data/pandamium/functions/triggers/dimension_count.mcfunction

```diff
@@ -0,0 +1,14 @@
+execute store result score <overworld_player_count> variable if entity @a[nbt={Dimension:"minecraft:overworld"}]
+execute store result score <nether_player_count> variable if entity @a[nbt={Dimension:"minecraft:the_nether"}]
+execute store result score <end_player_count> variable if entity @a[nbt={Dimension:"minecraft:the_end"}]
+
+tellraw @s {"text": "==== Dimension Count ====","color":"aqua","bold":true}
+
+tellraw @s [{"text": "Overworld | ","color":"gray"},{"score":{"name":"<overworld_player_count>","objective":"variable"},"color":"gray"}]

```

Comment:
Could you use the same format as the other menus, please: green and aqua colours scheme (see `misc/get_playtime`)